src/components/stylingGuide.stories.mdx

```diff
@@ -0,0 +1,110 @@
+<Meta title="Styling Guide" />
+
+# Styling Guide
+
+The Amazon Chime SDK React Component Library is comprised of Providers, Hooks, UI Components, and SDK-connected components. 
+Each component has some basic styling applied to it, but you can add or override your own styles to the component by using stylesheets,
+`styled-components` or other styling frameworks. 
+
+The Amazon Chime SDK React Component Library uses [`styled-components`](https://styled-components.com/docs) to attach base styles to all of the components. You must install the peer dependency on `styled-components` to use the library. 
+
+
+# Base Styles
+Every component extends either the [BaseSdkProps](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/61464e15175b0f88d141f968790524c5443ea5d1/src/components/sdk/Base/index.tsx) props or [BaseProps](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/61464e15175b0f88d141f968790524c5443ea5d1/src/components/ui/Base/index.ts), which include the `css` and `classname` props. 
+
+# Overriding styles on a component
+There are multiple ways to override the base styling applied on SDK or UI components.
+
+Each UI or SDK component contains one or more dom elements, and in some cases the dom elements are nested within each other or dynamically rendered.
+In order select these dom elements in your stylesheets, you can refer to the individual component's documentation.
+
+## 1. Override styles using the `css` prop
+The first method is by providing the `css` prop to any of the components. This prop is meant to be used
+for minor changes and is not recommended if making more than a couple styling changes for a component.
+
+```
+// CSS prop 
+<Badge css="border: solid 1px black"/>
+```
+
+## 2. Override styles using the `className` prop
```

Comment:
nit: How about moving this option to #1 since it is the preferred method.

Reply:
sure, will make this last suggestion